SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1506 @@
+			$notificationThresholdInDays = $products.productAttributes.passwdExpReminderInDay
+			$notificationIntervalInDays = $products.productAttributes.passwdExpReminderNotificationFrequencyInDay
+			$numAttempts = $products.productAttributes.passwdMaxFailAttempts
+			$attemptIntervalInMins = $products.productAttributes.passwdAttemptsIntervalInMins
+			$unlockIntervalInMins = $products.productAttributes.passwdUnlockIntervalInMins
+			Write-LogMessage -type INFO -Message "Workspace ONE Access: Setting the password policy."
+			Start-Sleep -s 10
+			Request-WSAToken -fqdn $ppmVariables.wsaFqdn -user $ppmVariables.wsaAdminUser -pass $ppmVariables.wsaAdminPass | Out-Null
+			Set-WSAPasswordPolicy -minLen $minCharacterLength -minLower $minLowercaseChar -minUpper $minUppercaseChar -minDigit $minNumericChar -minSpecial $minSpecialChar -history $passwdHistory -maxConsecutiveIdenticalCharacters $maxConsecutiveIdenticalChar -maxPreviousPasswordCharactersReused $maxPreviousPasswordCharactersReused -tempPasswordTtlInHrs $tempPasswordTtlInHrs -passwordTtlInDays $passwordTtlInDays -notificationThresholdInDays $notificationThresholdInDays -notificationIntervalInDays $notificationIntervalInDays | Out-Null
+			Set-WSAPasswordLockout -numAttempts $numAttempts -attemptInterval $attemptIntervalInMins -unlockInterval $unlockIntervalInMins | Out-Null
+			Write-LogMessage -type INFO -Message "Workspace ONE Access: Completed setting the password policy."
+		}
+	} else {
+		Write-LogMessage -Type INFO -Message "Workspace ONE Access skipped; login details not provided."  
+	}
+}
+
+# MAIN FUNCTION
+Try {
+	if ((($publishHTML -or $publishJSON) -and $applyPasswordPolicy) -or ($publishHTML -and $publishJSON)) {
+		Write-LogMessage -Type ERROR -Message "Error multiple options triggered.  Please request only one option -publishJSON, -publishHTML or -applyPasswordPolicy." -Colour Red
+		Exit
+	} elseif ($publishJSON -and $ppmVariables.skipDocComparison -eq $false -and $driftOnly) {
+		importStandardConfigurations -ppmVariables $ppmVariables -ppmStandardConfigValues $ppmStandardConfigValues -ppmEnvironmentalDetails $ppmEnvironmentalDetails
+		getEnvironmentPasswordPolicyDetail -ppmVariables $ppmVariables -ppmEnvironmentalDetails $ppmEnvironmentalDetails
+		exportReportJSON -ppmVariables $ppmVariables -ppmStandardConfigValues $ppmStandardConfigValues -ppmEnvironmentalDetails $ppmEnvironmentalDetails
+		Exit
+	} elseif ($publishJSON -and $ppmVariables.skipDocComparison -eq $false -and $driftOnly -eq $false) {
+		importStandardConfigurations -ppmVariables $ppmVariables -ppmStandardConfigValues $ppmStandardConfigValues -ppmEnvironmentalDetails $ppmEnvironmentalDetails
+		getEnvironmentPasswordPolicyDetail -ppmVariables $ppmVariables -ppmEnvironmentalDetails $ppmEnvironmentalDetails
+		exportReportJSON -ppmVariables $ppmVariables -ppmStandardConfigValues $ppmStandardConfigValues -ppmEnvironmentalDetails $ppmEnvironmentalDetails  -doComparison
```

Comment:
The param is `docCompare` currently.

Reply:
addressed in 7c76aadcc899391eebaf87062fd4f1e38b005834